Experiments/Experiments/DefaultFeatureFlagService.swift

```diff
@@ -91,6 +91,8 @@ public struct DefaultFeatureFlagService: FeatureFlagService {
             return buildConfig == .localDeveloper || buildConfig == .alpha
         case .splitViewInProductsTab:
             return false
+        case .useURLSessionInWordPressKit:
+            return buildConfig != .appStore
```

Comment:
Did you consider making this a remote flag?

Reply:
No, that didn't cross my mind. I don't see the new URLSession thing as a feature that we'd like to remotely turn on or off though.

Reply:
Good point. The only advantage I see would be in the short transition phase, if we find something tragically wrong in production. But... given how deep and shared the `URLSession` usages are, it seems unlikely something would go unnoticed during testing.